### PR TITLE
improve testing docs

### PR DESCRIPTION
**Pull request by richm**
improve testing docs, especially to help first time test writers

### CODE REVIEW

**Comment by richm** [comment]
@pcahyna @marmolak please review

File: README-testing.md [hunk 1]
```diff
@@ -63,19 +63,93 @@ Where `args.json` looks like this:
...
+## Local Virtual Machine based tests
+
+### Setup and running
+
+You must first install the commands
+`/usr/share/ansible/inventory/standard-inventory-qcow2` and `/usr/bin/ansible-playbook`.  On Fedora, this is
+provided by the `standard-test-roles-inventory-qemu` and `ansible` packages -
+`sudo dnf -y install standard-test-roles-inventory-qemu ansible`.
+The test playbooks are the files
+that match the pattern `tests/tests_*.yml`.  You must also have one or more
```
Comment (pcahyna) [inline README-testing.md:86]: it is `tests/tests*.yml` : https://docs.fedoraproject.org/en-US/ci/tests/#_enabling

**Comment by nhosoi** [comment]
lgtm (Now I want to refacter logging tests one more time based on this doc... :)
I mentioned TEST_DEBUG option in the logging test readme. Isn't it useful for the test developer here?

**Comment by richm** [comment]
> lgtm (Now I want to refacter logging tests one more time based on this doc... :)
> I mentioned TEST_DEBUG option in the logging test readme. Isn't it useful for the test developer here?

Yes, it is.

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

**Pull request closed without merging**